duckchat/duckchat-impl/src/main/java/com/duckduckgo/duckchat/impl/helper/DuckChatJSHelper.kt

```diff
@@ -46,6 +46,7 @@ class RealDuckChatJSHelper @Inject constructor(
     private val duckChatPixels: DuckChatPixels,
     private val dataStore: DuckChatDataStore,
 ) : DuckChatJSHelper {
+    private val migrationItems = mutableListOf<String>()
```

Comment:
@joshliebe is this ok here? the data is not designed to stick around, so this seemed ok - but please let me know otherwise!

Reply:
yep, just a string